wg-naming/recommendations/002-blacklist-whitelist-allowlist-denylist.md

```diff
@@ -0,0 +1,95 @@
+* Is the term overtly sexist, transphobic, or pejorative about a gender identity? **No**
+* Is the term overtly ableist, or pejorative to neurodiverse or disabled people? **No**
+* Is the term overtly homophobic? **No**
+
+
+**Second order concerns**:
+
+In short: is the term ambigously harmful, or is it harmful but not to a specific identity? **Yes**. Once again, see [this article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/)) 
+
+In particular, see the following pull-quote:
+
+> In this context, it is worth examining the origins of the term “blacklist” from the Douglas Harper Etymology Dictionary, which states that its origin and history is:
+
+ > >  n.
+
+ > >  also black-list, black list, “list of persons who have incurred suspicion,” 1610s, from black (adj.), here indicative of disgrace, censure, punishment (attested from 1590s, in black book) + list (n.). Specifically of employers’ list of workers considered troublesome (usually for union activity) is from 1888. As a verb, from 1718. Related: Blacklisted; blacklisting. [32]
+
+> It is notable that the first recorded use of the term occurs at the time of mass enslavement and forced deportation of Africans to work in European-held colonies in the Americas.
+
+While it is not directly attacking specific identities, in other context it does have negative connotations.
+
+* Is the term violent? **Partially**
+* Is the term milataristic? **No**
+
+**Third-order concerns**
+
+* Is the term evocative instead of descriptive? **Partially** 
+* Is the term ambiguous? **Yes. As mentioned in the above description, "black" and "white" have different meanings in different cultures.**
+
+
+## Precedents 
+
+* [“Blacklists” and “whitelists”: a salutary warning concerning the prevalence of racist language in discussions of predatory publishing](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/)
+
+* [IETF Network Working Group: Terminology, Power and Oppressive Language](https://tools.ietf.org/id/draft-knodel-terminology-00.html)
+
+* [Android issue changing these terms (screenshot)](https://9to5google.com/wp-content/uploads/sites/4/2020/06/android-aosp-allowlist-explanation.png)
+
+* [Twitter's language changes](https://www.cnet.com/news/twitter-engineers-replace-racially-loaded-tech-terms-like-master-slave/)
+
+## Impact
+
+The majority of uses of blacklist and whitelist occur in `/vendor` directories.
+
+* [Hound search - blacklist](https://cs.k8s.io/?q=blacklist&i=nope&files=&repos=)
+* [Hound search - whitelist](https://cs.k8s.io/?q=whitelist&i=nope&files=&repos=)
+
+If we exclude these, areas for replacement narrow. Many repositories have relatively minor uses outside `/vendor` paths, but the following are significant:
+
+* [kubernetes/kops](https://github.com/kubernetes/kops/): Contains breaking changes (`-whitelisted-healthcheck-cidr` flag, among other uses). Xontains uses both in documentation & code.
```

Comment:
tiny typo
```suggestion
* [kubernetes/kops](https://github.com/kubernetes/kops/): Contains breaking changes (`-whitelisted-healthcheck-cidr` flag, among other uses). Contains uses both in documentation & code.
```

Reply:
I am a little confused with the statement formation. Not sure which is correct. But does the following read more easily than the one in the doc? 

`Contains usages in both documentation & code.`

Reply:
Yup, it's a typo.